templates/event.php

```diff
@@ -73,6 +73,12 @@
 		</table>
 	</div>
 <?php endif ?>
+<div>
+	<details class="event-stats-summary">
+		<summary>View stats summary in text </summary>
+		<p class="event-stats-text"><?php echo esc_html( sprintf( 'At the %s event, %d people contributed in %d languages (%s), translated %d strings and reviewed %d strings.', esc_html( $event_title ), esc_html( $event_stats->totals()->users ), count( $event_stats->rows() ), esc_html( implode( ',', array_keys( $event_stats->rows() ) ) ), esc_html( $event_stats->totals()->created ), esc_html( $event_stats->totals()->reviewed ) ) ); ?></p>
```

Comment:
We could consider breaking this line up as it's really long, which makes it hard to read, IMHO.